ui/hooks/useTokensToSearch.js

```diff
@@ -119,7 +119,12 @@ export function useTokensToSearch({ usersTokens = [], topTokens = {} }) {
       others: [],
     };
 
-    memoizedTokensToSearch.forEach((token) => {
+    const memoizedSwapsAndUserTokensWithoutDuplicities = uniqBy(
+      [...memoizedTokensToSearch, ...memoizedUsersToken],
+      'symbol',
```

Comment:
I think we should use the `address` as the property to determine uniqueness, because different tokens can share symbols.